Assets/BlockModel.cs

```diff
@@ -1,5 +1,5 @@
 ﻿
-public struct BlockModel {
+public class BlockModel {
```

Comment:
move to class, I was copying it too much around.
And I know it's immutable, I'll make the setter private next time